src/issues/components/IssueResults.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <QCard>
+    <QCardTitle>
+      {{ $t('ISSUE.VOTING.RESULTS.TIME_UP') }}
+    </QCardTitle>
+    <QCardMain>
+      <div>
+        <strong
+          class="q-mr-sm"
+        >
+          {{ $t('ISSUE.VOTING.RESULTS.ENDED_AT', { date: $d(issue.votings.expiresAt, 'long') }) }}
+        </strong>
+      </div>
+      <div
+        class="q-pt-xs"
+      >
+        {{ $t('ISSUE.VOTING.RESULTS.PARTICIPANTS', { number: lastDecision.participantCount }) }}
+      </div>
+      <QItem>
+        <small class="text-weight-light">
+          <span>
+            {{ $t('ISSUE.INITIATED_BY', { initiatorName: issue.createdBy.displayName }) }}
+            <DateAsWords :date="issue.createdAt" />
+          </span>
+        </small>
+        <QItemMain/>
+        <QItemSide
+          v-if="issue.status !== 'cancelled'"
+          right
+          stamp="Total score"
+        />
+      </QItem>
+      <QItem
+        v-if="issue.status === 'cancelled'"
+      >
+        <QItemMain>
+          <strong>
+            {{ $t('ISSUE.VOTING.RESULTS.UNDECIDED') }}
+          </strong>
+          <QItemTile
+            class="q-pt-sm"
+          >
+            {{ $t('ISSUE.VOTING.RESULTS.UNDECIDED_WHY') }}
+            <ul>
+              <li>
+                {{ $t('ISSUE.VOTING.RESULTS.UNDECIDED_REASON_1') }}
+              </li>
+              <li>
+                {{ $t('ISSUE.VOTING.RESULTS.UNDECIDED_REASON_2') }}
+              </li>
+            </ul>
+          </QItemTile>
+        </QItemMain>
+      </QItem>
+      <QList
+        v-if="issue.status !== 'cancelled'"
+        no-border
+      >
+        <QItem
+          v-for="(o, index) in sortedArray"
+          :key="o.id"
+        >
+          <QItemSide
+            :icon="getIcon(index)"
+            color="primary"
+          />
+          <QItemMain>
+            {{ displayOutcomes(index) }}
+          </QItemMain>
+          <QItemSide>
+            {{ sortedArray[index].sumScore }}
+          </QItemSide>
+        </QItem>
+      </QList>
+    </QCardMain>
+  </QCard>
+</template>
+
+<script>
+import DateAsWords from '@/utils/components/DateAsWords'
+import max from 'date-fns/max'
+
+import {
+  QCard,
+  QCardMain,
+  QCardTitle,
+  QList,
+  QItem,
+  QItemSide,
+  QItemMain,
+  QItemTile,
+} from 'quasar'
+
+export default {
+  components: {
+    QCard,
+    QCardMain,
+    QCardTitle,
+    QList,
+    QItem,
+    QItemSide,
+    QItemMain,
+    QItemTile,
+    DateAsWords,
+  },
+  props: {
+    issue: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    lastDecision () {
+      return this.issue.votings.find(v => max(v.expiredAt))
```

Comment:
I think this is not doing what it should do. [max](https://date-fns.org/v1.9.0/docs/max) expects an array and returns the latest date. `find` will give you the first list entry with a `expiredAt` set, so essentially it's the same as `this.issue.votings[0]`.

We want to find the latest date, so we could sort the array by expiredAt and take the first entry:

```
votings.slice().sort((a, b) => b.expiredAt - a.expiredAt)[0]
```
(I hope the sort order is right...)

Although my code would fail if there are no votings. That should not happen, but to failproof our code, we could add a check before.